docs/source/getting-started.mdx

```diff
@@ -113,16 +120,20 @@ If you are using JavaScript, create a `index.js` file that will contain **all**
   touch index.js
 ```
 
-Then add the following script to your `package.json`:
-
+Now replace the default `scripts` entry in your `package.json` file with these `type` and `scripts` entries:
 ```json title="package.json"
 {
+  // ...etc.
+  "type": "module",
   "scripts": {
-    "start": "node index.js"
+    "start": "tsc && node ./dist/index.js"
```

Comment:
We shouldn't need to compile with TS here, so this should be:
```suggestion
    "start": "node index.js"
```

Reply:
Oops, good catch! (We do need the `type` line though, or we could name the file index.mjs.)